mslib/msui/multilayers.py

```diff
@@ -197,10 +197,13 @@ def reload_sync(self):
         if self.synced_reference.itime not in self.synced_reference.itimes:
             self.synced_reference.itime = itimes[-1] if itimes else None
 
-        if self.synced_reference.vtime not in self.synced_reference.vtimes or \
-                self.synced_reference.vtime < self.synced_reference.itime:
-            self.synced_reference.vtime = next((vtime for vtime in vtimes if
-                                                vtime >= self.synced_reference.itime), None) if vtimes else None
+        if self.synced_reference.itime is not None:
+            if (self.synced_reference.vtime not in self.synced_reference.vtimes or
+                    self.synced_reference.vtime < self.synced_reference.itime):
```

Comment:
for me this is not easy to read. I think it gets better if there is a comment describing this

Reply:
I tried to make it more readable by reducing visual clutter and the ternary expressions.